executor/executor.go

```diff
@@ -63,6 +66,16 @@ func New(sdAPI api.API, args []string) (Executor, error) {
 
 func execCommand(path string, args []string) (err error) {
 	cmd := command(path, args...)
+	if !terminal.IsTerminal(0) {
+		stdin, err := cmd.StdinPipe()
+		if nil != err {
+			lgr.Debug.Printf("failed to open StdinPipe: %v", err)
+		}
+		b, _ := ioutil.ReadAll(os.Stdin)
+		io.WriteString(stdin, string(b))
```

Comment:
WriteString return error, so I think you also need check this error.

Reply:
Added error handle.